src/shared/ui/components/BaseTable.vue

```diff
@@ -86,15 +86,16 @@ import { computed } from 'vue';
 import { useWindowSize } from '@vueuse/core';
 import ArrowIcon from '@/shared/ui/icons/arrow.svg';
 import BaseLoading from './BaseLoading.vue';
-import type { TablePagination } from '@/shared/lib/table';
 import BaseDropdown from '@/shared/ui/components/BaseDropdown.vue';
+import type { Pagination } from '@/shared/api/dto';
 
 interface Props {
   loading: boolean
-  pagination?: TablePagination | null
+  pagination?: Pagination | null
   items: T[]
   containerClass: string
   breakpoint?: string | number
+  sticky?: boolean
```

Comment:
What _sticky_ means? Can it be a name that describes the behaviour better?

Reply:
I have already renamed it to reversed✅